tests/foreman/ui/test_org.py

```diff
@@ -405,33 +409,38 @@ def test_remove_user(self, user_name):
         """
         strategy, value = common_locators['entity_select']
         strategy1, value1 = common_locators['entity_deselect']
-        org_name = gen_string('alpha')
-        user = entities.User(
-            login=user_name,
-            firstname=user_name,
-            lastname=user_name,
-            password=gen_string('alpha'),
-        ).create()
-        self.assertEqual(user.login, user_name)
         with Session(self.browser) as session:
-            make_org(session, org_name=org_name, users=[user_name])
-            self.org.search(org_name).click()
-            session.nav.click(tab_locators['context.tab_users'])
-            element = session.nav.wait_until_element((strategy1,
-                                                      value1 % user_name))
-            # Item is listed in 'Selected Items' list and not 'All Items' list.
-            self.assertIsNotNone(element)
-            self.org.update(org_name, users=[user_name], new_users=None)
-            self.org.search(org_name).click()
-            session.nav.click(tab_locators['context.tab_users'])
-            element = session.nav.wait_until_element((strategy,
-                                                      value % user_name))
-            # Item is listed in 'All Items' list and not 'Selected Items' list.
-            self.assertIsNotNone(element)
+            for user_name in valid_users():
```

Comment:
Same comment as before about creating only one organization and then adding/removing users. Not a blocker for this PR.